docs/farming-&-staking/farming/space-acres/_category_.json

```diff
@@ -1,8 +1,5 @@
 {
-    "label": "Space Acres (Recommended)",
-    "position": 2,
-    "link": {
-      "type": "generated-index",
-      "description": "Autonomys GUI Application Space Acres"
-    }
+    "label": "✨ Space Acres",
```

Comment:
Unfortunately, ✨ meaning might not be clear for some users, I think we have to keep **(recommended)** for both the SubWallet and Space Acres. 

I like the idea though

Reply:
Can you give me your thoughts on this new approach: Added a floating label below the sidebar to indicate recommended applications. This should better clarify the purpose of the stars without needing to use `(recommended)` in the navigation. This also allows for future applications and guides to be easily recommended in a cleaner fashion.

Reply:
I still think you have to know where to look to see the label explaining what the ✨ means. 
I don't mind giving this approach a try and seeing if there's any user feedback about it.